packages/runner/lib/app.ts

```diff
@@ -15,7 +15,7 @@ try {
     });
 
     const close = () => {
-        logger.info('Closing...');
+        logger.info(`${id} Closing...`);
```

Comment:
unrelated to this PR but it is nice when running Nango locally to understand which runner is being terminated

Reply:
what do you mean by `create a child`?